README.md

```diff
@@ -30,27 +30,27 @@ I run a very simple multi-process benchmark, with 5 iterations to get a median o
 Operations per millisecond (*higher is better*):
 
 
-| name                                                           | set   | get1  | update | get2  | evict |
-|----------------------------------------------------------------|-------|-------|--------|-------|-------|
-| [lru_cache](https://npmjs.com/package/lru_cache)               | 10390 | 30864 | 8598   | 29112 | 12173 |
-| [lru-cache](https://npmjs.com/package/lru-cache)               | 4468  | 5593  | 4409   | 5907  | 7943  |
-| [lru-fast](https://npmjs.com/package/lru-fast)                 | 3492  | 20513 | 19342  | 34072 | 7286  |
-| [simple-lru-cache](https://npmjs.com/package/simple-lru-cache) | 3956  | 29369 | 13860  | 27855 | 6502  |
-| [tiny-lru](https://npmjs.com/package/tiny-lru)                 | 7112  | 14706 | 29630  | 24907 | 4496  |
-| [hashlru](https://npmjs.com/package/hashlru)                   | 3112  | 4765  | 4377   | 4726  | 4196  |
-| [quick-lru](https://npmjs.com/package/quick-lru)               | 3584  | 3257  | 4338   | 3609  | 3893  |
-| [hyperlru-object](https://npmjs.com/package/hyperlru-object)   | 2167  | 7353  | 6913   | 12195 | 2519  |
-| [lru](https://www.npmjs.com/package/lru)                       | 1757  | 4381  | 4077   | 4840  | 1807  |
-| [js-lru](https://www.npmjs.com/package/quick-lru)              | 1419  | 6180  | 7223   | 10477 | 1508  |
-| [secondary-cache](https://npmjs.com/package/secondary-cache)   | 1268  | 4420  | 3368   | 7663  | 1371  |
-| [hyperlru-map](https://npmjs.com/package/hyperlru-map)         | 1149  | 3927  | 5276   | 9355  | 1206  |
-| [modern-lru](https://npmjs.com/package/modern-lru)             | 839   | 4542  | 3799   | 4689  | 939   |
-| [mkc](https://npmjs.com/packacge/package/mkc)                  | 667   | 1528  | 984    | 1734  | 621   |
+| name                                                           | set  | get1  | update | get2  | evict |
+|----------------------------------------------------------------|------|-------|--------|-------|-------|
+| [lru_cache](https://npmjs.com/package/lru_cache)               | 3436 | 19084 | 5105   | 19417 | 9425  |
+| [tiny-lru](https://npmjs.com/package/tiny-lru)                 | 5993 | 20790 | 21692  | 18797 | 5157  |
+| [simple-lru-cache](https://npmjs.com/package/simple-lru-cache) | 3204 | 24783 | 16639  | 40984 | 4962  |
+| [lru-cache](https://npmjs.com/package/lru-cache)               | 2096 | 2694  | 2901   | 2978  | 4183  |
+| [quick-lru](https://npmjs.com/package/quick-lru)               | 2601 | 2510  | 2764   | 2742  | 3342  |
+| [hashlru](https://npmjs.com/package/hashlru)                   | 5222 | 6135  | 3039   | 3656  | 2751  |
+| [lru-fast](https://npmjs.com/package/lru-fast)                 | 2157 | 28612 | 17905  | 31104 | 2650  |
+| [hyperlru-map](https://npmjs.com/package/hyperlru-map)         | 1971 | 7728  | 5663   | 8396  | 2282  |
```

Comment:
Hey @avoidwork, this name is wrong, `hyperlru-object` is declared at line 46

Reply:
https://github.com/dominictarr/bench-lru/pull/24